### ISSUE TITLE
Screen API not working under Windows.

### ISSUE THREAD

**Issue by delvedor** [electron/electron#5034]
Hi, I'm trying to use *getAllDisplays()* API under Windows and I have a very strange behaviour.

Code:
```javascript
const screen = require('electron').screen
const arr = screen.getAllDisplays()
console.log(arr)
```

Use case:
1) Start the electron app
2) Call the above code
3) I get the monitor array in the output
4) I change the resolution/disable external monitor/etc
5) I re run the code of above (**without relaunching** the electron app)
6) I obtain the same output of the point 3), nothing changed. (in some case I get duplicate monitors)

Under OSX there's no problem.
I guess this problem is related to #3075.

In conclusion, work with the screen API under Windows is basically impossible.

* Electron version: 0.37.4
* Operating system: Windows

**Comment by delvedor** [comment]
I've written [this](https://github.com/delvedor/electron-screen) module that gives me the current (and correct) display bound data.

**Comment by zcbenz** [comment]
I'm merging this to #3075 since they are essentially the same bug.
